sources/packages/backend/libs/sims-db/src/entities/education-program.model.ts

```diff
@@ -259,9 +259,9 @@ export class EducationProgram extends RecordDataModel {
    */
   @Column({
     name: "none_of_entrance_requirements",
-    nullable: true,
+    nullable: false,
```

Comment:
Are the other columns from the same set of options also setting nullable `false`?

Reply:
No, they are optional and nullable: 
![image](https://github.com/bcgov/SIMS/assets/78114138/20ae4ea7-d2d1-44d1-9112-d78d3477eced)

Should we update them too and make them mandatory?

Reply:
I believe that we should go one way or another. Either have all of them changed or keep the PR simple and do not change the `none_of_entrance_requirements`, unless it would result in an issue. Would it be an issue not making the column mandatory right now?

I assume that it was an optional field in the past and then it became mandatory.
If we make them all mandatory we just need how complex would be change.
I guess that the EntranceRequirements models are already expecting a not null value.
```ts
export class EntranceRequirements {
  @Expose()
  hasMinimumAge: boolean;
  @Expose()
  minHighSchool: boolean;
  @Expose()
  requirementsByInstitution: boolean;
  @Expose()
  requirementsByBCITA: boolean;
  @Expose()
  noneOfTheAboveEntranceRequirements: boolean;
}
```

Reply:
We don't need to make `none_of_entrance_requirements` NOT NULL. The formio change will be enough to fix the bug. I'm going to revert the changes to DB.